FusionMCPBridge/handlers/design/geometry_impl.py

```
@@ -0,0 +1,324 @@
+            bodies.add(latest_body)
+        else:
+            ui.messageBox("Keine Bodies gefunden.")
+            return
+
+        vector = adsk.core.Vector3D.create(x, y, z)
+        transform = adsk.core.Matrix3D.create()
+        transform.translation = vector
+        moveFeatureInput = moveFeats.createInput2(bodies)
+        moveFeatureInput.defineAsFreeMove(transform)
+        moveFeats.add(moveFeatureInput)
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)